src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -333,7 +415,11 @@ private void deContributeInstance(UUID centralServerId, Instance instance, Stati
       stats.addRecordsTotal(1);
       recordContributionService.deContributeInstance(centralServerId, instance);
       stats.addRecordsDeContributed(1);
-    } catch (Exception e) {
+    }
+    catch (ServiceSuspendedException ex) {
+      throw new ServiceSuspendedException(ex.getMessage());
+    }
+    catch (Exception e) {
       instanceExceptionListener.logWriteError(e, instance.getId());
     } finally {
```

Comment:
You're not incrementing `recordsProcessed` here. Is that because you feel `contributeItemsChunk` will handle the incrementation?

Reply:
this is part of the **deContrbution** which in this PR or in this story not handled. This PR is all about initial contribution.